helios-services/src/main/java/com/spotify/helios/agent/HostInfoReporter.java

```diff
@@ -93,6 +96,10 @@ protected void runOneIteration() throws InterruptedException {
         .build();
 
     nodeUpdater.update(hostInfo.toJsonBytes());
+
+    // Touch the host ID node in ZK so that later on AgentZooKeeperRegistrar can know how long a
+    // host has been down.
+    //hostIdUpdater.touch();
```

Comment:
Adding this breaks the tests. @gimaker @rohansingh @mattnworb maybe you could take a second look with me when you guys get a chance.